source/reference/mongosync.txt

```diff
@@ -121,6 +121,14 @@ Behavior
 
 .. _c2c-mongosync-config:
 
+Cluster Independence
+~~~~~~~~~~~~~~~~~~~~
+
+``mongosync`` synchronizes collection data between a source cluster and
```

Comment:
"between a source and destination cluster"

Reply:
Done